python/taichi/lang/matrix.py

```diff
@@ -898,10 +898,10 @@ def any(self):
             >>> v.any()
             True
         """
-        ret = ops_mod.cmp_ne(self.entries[0], 0)
-        for i in range(1, len(self.entries)):
```

Comment:
Rename `i` to `e` or something?

Reply:
I guess `elem`

Reply:
Maybe `entry` is better suited in this file

Reply:
You're right